docs/source/ephys_at_swc/example_pipelines/examples/README.rst

```diff
@@ -0,0 +1,39 @@
+:orphan:
+
+Python Pipelines
+================
+
+Multiple probes (Cambridge Neurotech)
+-------------------------------------
+
+Dammy Onih (Akrami lab) is running a multi-session paradigm with two
+`Cambridge Neurotech <https://www.cambridgeneurotech.com/neural-probes>`__
+probes. In this multi-session task, mice learn a statistical learning paradigm to a
+reward-associated auditory stimulus, recording from the hippocampus.
+The pipeline uses SpikeInterface for preprocessing,
+sorting, and analysis and can be found `here <https://github.com/AOONIH/ephys/tree/master>`__.
+
+The IBL analysis pipeline
+-------------------------
+
+Nate Miska (Mrsic-Flogel lab) is a member of the
+`International Brain Laboratory
+(IBL) <https://www.internationalbrainlab.com/>`_
+running the
+`IBL's standardised behavioural task <https://elifesciences.org/articles/63711>`_
+with acute Neuropixels 1.0 recordings. Details of the
+`analysis pipeline code <https://github.com/int-brain-lab/ibl-neuropixel>`__
+on the IBL data management system can be found
+`here <https://int-brain-lab.github.io/iblenv/index.html>`_.
+
+Integrated analysis with Neuropixels 2.0
+----------------------------------------
+
+Steve Lenzi (Margrie Lab) has an integrated pipeline for automated behavioural,
+electrophysiological, and anatomical analysis. Steve works with a
+multi-session escape paradigm, recording stimulus-responsive neurons in the posterior
+striatum. He is using NeuroPixels 2.0 and SpikeGLX with a SpikeInterface-based
+pipeline; the code can be found `here <https://github.com/stephenlenzi/npix_lse>`__.
+
+Python Scripts
```

Comment:
It's not immediately clear how the scripts section is different to the other parts above.

Reply:
"Standalone scripts"?